source/renderer/app/components/wallet/WalletCreateDialog.js

```diff
@@ -211,9 +211,19 @@ export default class WalletCreateDialog extends Component<Props, State> {
     const { isSubmitting } = this.state;
     const dialogClasses = classnames([styles.component, 'WalletCreateDialog']);
 
+    const walletNameField = form.$('walletName');
+    const spendingPasswordField = form.$('spendingPassword');
+    const repeatedPasswordField = form.$('repeatPassword');
+
+    const formValuesNotSet =
+      !walletNameField.value ||
+      !spendingPasswordField.value ||
+      !repeatedPasswordField.value;
+
     const actions = [
       {
         className: isSubmitting ? styles.isSubmitting : null,
+        disabled: isSubmitting || formValuesNotSet,
```

Comment:
@DeeJayElly Replace this with `canSubmit` once the variable is renamed

Reply:
Done